.github/CODEOWNERS

```
@@ -8,4 +8,4 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-* @heyams @trask
\ No newline at end of file
+* @heyams @trask @jeanbisutti
```

just to be alphabetical

```suggestion
* @heyams @jeanbisutti @trask
```